versioned/spi/src/main/java/org/projectnessie/versioned/VersionStore.java

```diff
@@ -208,6 +194,22 @@ Hash create(NamedRef ref, Optional<Hash> targetHash)
   void delete(NamedRef ref, Optional<Hash> hash)
       throws ReferenceNotFoundException, ReferenceConflictException;
 
+  /**
+   * Provide the current hash for the given NamedRef.
```

Comment:
maybe `Resolve the current hash for the given NamedRef`?

Reply:
changed the comment